js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedTable.tsx

```diff
@@ -195,7 +195,7 @@ export const RunsFeedTableWithFilters = ({
   filter: RunsFilter;
 } & Pick<
   RunsFeedTableProps,
-  'actionBarComponents' | 'belowActionBarComponents' | 'emptyState' | 'hideTags'
+  'actionBarComponents' | 'belowActionBarComponents' | 'emptyState' | 'hideTags' | 'scroll'
```

Comment:
you're not actually passing scroll on line 221, it's hardcoded to false still

Reply:
ahh It's overridden in `...rest` actually, but i'll make it explicit

Reply:
Updated this to destructure it and apply the default value as `?? false`

Reply:
oops, missed that

Reply:
I think what you had before is fine